ui/packages/catalog-ui-search/src/main/webapp/react-component/auto-complete/auto-complete.js

```diff
@@ -99,4 +100,28 @@ class AutoComplete extends React.Component {
   }
 }
 
+AutoComplete.defaultProps = {
+  minimumInputLength: 3,
+}
+
+AutoComplete.propTypes = {
+  /** Default placeholder before user has selected an item. */
+  placeholder: PropTypes.node,
+
+  /** The current selected value. */
+  value: PropTypes.any,
```

Comment:
Is this needed just for docgen?

Reply:
Yes. Adding proptypes allows us to generate documentation for components.